src/en/xoxocomics/src/eu/kanade/tachiyomi/extension/en/xoxocomics/XoxoComics.kt

```diff
@@ -15,7 +15,13 @@ import org.jsoup.nodes.Element
 import java.text.SimpleDateFormat
 import java.util.Locale
 
-class XoxoComics : WPComics("XOXO Comics", "https://xoxocomic.com", "en", SimpleDateFormat("MM/dd/yyyy", Locale.US), null) {
+class XoxoComics : WPComics(
+    name = "XOXO Comics",
+    baseUrl = "https://xoxocomic.com",
+    lang = "en",
```

Comment:
I didn't meant named argument for these too since they are part of the convention. I should have been more clear